lldb/test/API/source-manager/artificial_location.cpp

```diff
@@ -0,0 +1,8 @@
+#include "artificial_location.h"
+
+int A::foo() {
+#line 0
```

Comment:
FYI there's some discussion about this test going on in https://github.com/llvm/llvm-project/pull/107849, in case you had some thoughts on Jeremy's points

Reply:
Thanks for the pointer. Quite a coincidence that I end up touching the same test just as Jaremy ends up breaking it :P